wiki/state.md

```
@@ -155,3 +155,80 @@ Behaviors:
+## Global Translation
+
+See [global-translation.md](./global-translation.md) for why this exists.
+
+- Keeps translation activations, per-collection schema config, and translated content.
+- Path: `/translations`
+
+| Name        | Data Type                        | Description                              |
+| ----------- | --------------------------------- | ----------------------------------------- |
+| activations | Hash of [Activation](#activation) | key = activation id, from `activateTranslation` |
+| schema      | Hash of [CollectionSchema](#collectionschema) | key = collection id |
+| docs        | [Collection](#collection-1)       | Translated documents, one entry per `{collection, docId, language}` currently translated |
```

markdownlint-cli2 (0.23.1)

[warning] Link fragments should be valid

(MD051, link-fragments)